src/jit/emit.h

```diff
@@ -578,13 +578,13 @@ class emitter
     struct instrDesc
     {
     private:
-#if defined(_TARGET_XARCH_) && !defined(LEGACY_BACKEND)
+#if (defined(_TARGET_XARCH_) || defined(_TARGET_ARM64_)) && !defined(LEGACY_BACKEND)
         // The assembly instruction
         instruction _idIns : 9;
-#else  // !defined(_TARGET_XARCH_) || defined(LEGACY_BACKEND)
+#else  // !(defined(_TARGET_XARCH_) || defined(_TARGET_ARM64_)) || defined(LEGACY_BACKEND)
         // The assembly instruction
         instruction _idIns : 8;
-#endif // !defined(_TARGET_XARCH_) || defined(LEGACY_BACKEND)
+#endif // !(defined(_TARGET_XARCH_) || defined(_TARGET_ARM64_)) || defined(LEGACY_BACKEND)
         // The format for the instruction
         insFormat _idInsFmt : 8;
```

Comment:
We only have about 80 instruct formats for Arm64, so we could reduce the number of bits here by one if we want

Reply:
There are 116 formats today (counting IF_DEF in emitfmtsarm64.h).  Adding intrinsic support may add more. 

I am happy either way.  I can revise the patch to take the `_idInsFmt` bit now and add it back later if needed.